src/consts/OSFs.ts

```diff
@@ -0,0 +1 @@
+export const OSFS = ["GB", "HN", "HB", "KR", "WF", "NASO"];
```

Comment:
I know this is the order they gave them in, but should HB and HN be switched?  I see putting NASO at the end (similar to "Other"), but should the others be alphabetical?